Dockerfile.tests

```diff
@@ -11,4 +11,4 @@ COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/
 COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/run-e2e-nrop-serial.sh /usr/local/bin
 COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/numacell /bin
 COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/pause /
-ENTRYPOINT ["/usr/local/bin/run-e2e-nrop-serial.sh"]
+ENTRYPOINT ["./entrypoint.sh"]
```

Comment:
this should be turned into `CMD`, to read
```
CMD ["/usr/local/bin/run-e2e-nrop-serial.sh"]
```

this is not strictly needed, but it help in the context of this change to clarify the intent, and make the flow more extensible.
Then, rather than add this wrapper, just create your script the way you do below, but just add it into `/usr/local/bin`

Next, the container image will still have the existing run-e2e-nrop-serial.sh as entry point, but you will be able to override it and use the must-gather script you added.

You will need to pass parameters anyway, but this way is IMO cleaner and surely simpler: less wrappers, less code around:
```
podman run -ti quay.io/openshift-kni/numaresources-tests /usr/local/bin/run-e2e-must-gather.sh
```